src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -10,6 +10,7 @@ import org.mockito.{ArgumentCaptor, MockitoSugar}
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+import sangria.relay.{DefaultConnection, Edge}
```

Comment:
`Edge` doesn't look like it's used here. And again, it's not one of yours but `Tables` underneath isn't  either.